src/test/java/com/jcabi/github/RtBlobsTest.java

```diff
@@ -50,10 +51,20 @@
  * @author Alexander Lukashevich (sanai56967@gmail.com)
  * @version $Id$
  * @checkstyle MultipleStringLiteralsCheck (100 lines)
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
```

Comment:
@tmjee let's add a `@todo` puzzle to re-design this class and remove switching off checkstyle here